### PR TITLE
Changed the casl/ability default error message

### PR DESCRIPTION
**Pull request by YathurshanT**
Changed the casl/ability default error message

### CODE REVIEW

File: backend/services/src/shared/casl/policy.guard.ts [hunk 1]
```diff
@@ -8,7 +14,9 @@ import { Action } from "./action.enum";
 import { CaslAbilityFactory, AppAbility } from "./casl-ability.factory";
 import { CHECK_POLICIES_KEY } from "./policy.decorator";
 import { PolicyHandler } from "./policy.handler";
-const { rulesToQuery } = require('@casl/ability/extra');
+const { rulesToQuery } = require("@casl/ability/extra");
+
+const unAuthErrorMessage = "This action is unauthorised";
```
Comment (palindaa) [inline backend/services/src/shared/casl/policy.guard.ts:19]: Don't we need to add this to localisation file?

File: backend/services/src/shared/casl/policy.guard.ts [hunk 2]
```diff
@@ -141,9 +154,14 @@ export const PoliciesGuardEx = (injectQuery: boolean, action?: Action, subject?:
...
-      return policyHandlers.every((handler) =>
-        this.execPolicyHandler(handler, ability, body),
+      const pHandler = policyHandlers.every((handler) =>
+        this.execPolicyHandler(handler, ability, body)
       );
+      if (pHandler) {
+        return pHandler;
+      } else {
+        throw new ForbiddenException(unAuthErrorMessage);
+      }
```
Comment (dhanushkaxep) [inline backend/services/src/shared/casl/policy.guard.ts:164]: if policyHandlers.length == 0 then it wont reach here. So I think we need handle this inside the policyHandlers.length == 0 block also. please check

File: backend/services/src/shared/casl/policy.guard.ts (hunk 1 above)
Comment (YathurshanT) [reply backend/services/src/shared/casl/policy.guard.ts:19]: @palindaa the scenarios we are getting this error is while only sending the req via swagger or postman right? so I didn't add that, I'll check again anyway I will add this to the localisation file

File: backend/services/src/shared/casl/policy.guard.ts (hunk 2 above)
Comment (YathurshanT) [reply backend/services/src/shared/casl/policy.guard.ts:164]: okay

**Review by dhanushkaxep — APPROVED** [review]

**Pull request merged**